content/03-reference/01-tools-and-interfaces/02-prisma-client/050-filtering.mdx

```diff
@@ -526,16 +524,7 @@ const result = await prisma.user.findMany({
 
 ## Case-insensitive filtering
 
-Case-insensitive filtering is a preview feature and [can be enabled for PostgreSQL](case-sensitivity#options-for-case-insensitive-filtering). The feature is not required for MySQL and MariaDB because these databases are case-insensitive by default. Add the highlighted feature flag to enable case-insensitive filtering capabilities:
-
-```prisma
-generator client {
-  provider = "prisma-client-js"
-|  previewFeatures = ["insensitiveFilters"]
-}
-```
-
-> Please [share your feedback](https://github.com/prisma/prisma/issues/3340) on how this feature works for you. We are interested in both positive and negative feedback, so we know if this feature is already ready for production! (If encounter any problems, please open a [new issue in the `prisma/prisma` repository](https://github.com/prisma/prisma/issues/new/choose)).
+Case-insensitive filtering [can be enabled for PostgreSQL](case-sensitivity#options-for-case-insensitive-filtering). The feature is not required for MySQL and MariaDB because these databases are case-insensitive by default. 
```

Comment:
"The feature is not required" feels odd. Do you mean "The feature is not available" ?